routes.js

```diff
@@ -1,58 +1,75 @@
-const { error } = require('console');
 const express = require('express');
-const router = express.Router();
+const createMyCustomApi = express.Router();
+const fs = require('fs');
 
-// Middleware
-router.use(express.json());
+// Fetch data from the members.json file
+const loadMembers = () => {
+    const jsonData = fs.readFileSync(__dirname + "/" + "members.json");
+        return JSON.parse(jsonData);
+}
 
-// Get method (Get all member)
-router.get('/getAllMembers', (req, res) => {
-  res.send(teamMembersArray);
-  if(!error) {
-      res.status(200);
-  }
-});
+// Update members.js file
+const saveMembers = (newData) => {
+    const stringifyData = JSON.stringify(newData);
+    fs.writeFileSync(__dirname + "/" + "members.json", stringifyData)
+}
```

Comment:
Great! To make current this file focused on `routes`. Could you please make a next step and extract that logic into a file named like `dataAccess.js` or `membersFileAdapter.js` or `membersDataAccess.js`?

Reply:
Done!